services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/KayttajaTunniste.kt

```diff
@@ -0,0 +1,52 @@
+package fi.hel.haitaton.hanke.permissions
+
+import fi.hel.haitaton.hanke.getCurrentTimeUTC
+import java.security.SecureRandom
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.Id
+import javax.persistence.OneToOne
+import javax.persistence.Table
+import kotlin.streams.asSequence
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+
+@Entity
+@Table(name = "kayttaja_tunniste")
+class KayttajaTunnisteEntity(
+    @Id val id: UUID,
```

Comment:
Hankekayttaja entity creates the id as follows:

@Id val id: UUID = UUID.randomUUID(),

Could the same approach be used here?

Reply:
The default value for HankeKayttaja.id wasn't used anywhere. I added the default value here and removed `id = UUID.randomUUID()` elsewhere in the code.